### PR TITLE
LOOKDEVX-1625 - Use proper API to set color space in USD

### PR DESCRIPTION
**Pull request by JGamache-autodesk**

### CODE REVIEW

File: lib/mayaUsd/ufe/UsdAttributeHolder.cpp [hunk 1]
```diff
@@ -406,6 +406,9 @@ Ufe::Value UsdAttributeHolder::getMetadata(const std::string& key) const
             }
 #endif
             return Ufe::Value(niceName);
+        } else if (key == SdfFieldKeys->ColorSpace) {
+            auto csValue = _usdAttr.GetColorSpace();
```
Comment (feldstj) [inline lib/mayaUsd/ufe/UsdAttributeHolder.cpp:410]: Can this be const?

File: lib/mayaUsd/ufe/UsdAttributeHolder.cpp [hunk 2]
```diff
@@ -433,9 +436,17 @@ Ufe::Value UsdAttributeHolder::getMetadata(const std::string& key) const
 
 bool UsdAttributeHolder::setMetadata(const std::string& key, const Ufe::Value& value)
 {
-    if (isValid())
+    if (isValid()) {
+        if (key == SdfFieldKeys->ColorSpace) {
+            if (!value.empty() && value.isType<std::string>()) {
+                AttributeEditRouterContext ctx(_usdAttr.GetPrim(), _usdAttr.GetName());
```
Comment (feldstj) [inline lib/mayaUsd/ufe/UsdAttributeHolder.cpp:442]: Can this be const?

**Review by feldstj — APPROVED** [review]
LGTM, just a couple of minor comments.

**Review by seando-adsk — APPROVED** [review]

**Review by feldstj — APPROVED** [review]

**Comment by JGamache-autodesk** [comment]
@seando-adsk Ready for merge.

**Pull request merged**